src/node.cc

```diff
@@ -3780,7 +3780,10 @@ static void ParseArgs(int* argc,
 
   unsigned int index = 1;
   bool short_circuit = false;
-  while (index < nargs && argv[index][0] == '-' && !short_circuit) {
+  while (index < nargs &&
+         // starts with dash but is not only a dash which is special
```

Comment:
I’d drop this commend, or change it to something like `// skip "-"`, because currently it described how it does stuff, not what it does, so it’s directly redundant to the code.